recipe/bld.bat

```diff
@@ -10,6 +10,7 @@ cmake -G "NMake Makefiles" ^
       -DCMAKE_BUILD_TYPE:STRING=Release ^
       -DBUILD_SHARED_LIBS=ON ^
       -DRC_ENABLE_GTEST=ON ^
+      -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment:
I really do not like this, but I have not been able to find a way to generate the `.lib` file unfortunately.

On conda-forge, no project depend on rapidcheck:
```
mamba repoquery whoneeds -c conda-forge rapidcheck         
```

ArcticDB only uses it for tests (like other projects).

Reply:
For reference, a discussion with [Le Chat](https://chat.mistral.ai/chat/07ded5ba-2521-410b-9a9c-3409aef03aeb).

I think that's the best option for this library for now.